mgmt/Cargo.toml

```diff
@@ -12,6 +12,7 @@ required-features = ["bolero"]
 
 [features]
 default = []
+fake-pci-as-netdevsim = ["interface-manager/netdevsim"]
```

Comment:
This is not used in this PR, is it?

Reply:
I think I only partially understand the question.

This feature would, if activated, cause the `interface-manager/netdevsim` feature to be activated.  I introduce it in this PR because allowing "fake" pci devices in a future PR is part of the objective.

The intended functionality does not yet exist.  I can delay the introduction of this feature until after netdevsim functionality is more developed if you prefer.

Reply:
> This feature would, if activated, cause the `interface-manager/netdevsim` feature to be activated.

I understand that, thanks. I wanted to make sure my understanding was correct and the feature is not used yet.

> The intended functionality does not yet exist. I can delay the introduction of this feature until after netdevsim functionality is more developed if you prefer.

That's OK, no need to defer if you know you'll use it soon.

We don't have docs for the different packages' features yet, but how about adding a one-line comment in the Cargo.toml to at least summarise what this feature does?